### DISCUSSION TITLE
Can't select specific units in the editor, can't find them in the unit search menu

### DISCUSSION THREAD

**Discussion by AtomykAU** [diesel-modding/BeardLib-Editor#736] [Q&A] [answered]
I'm trying to remove units from certain areas of a level so that I can take wide screenshots of the areas behind them to use in developing a 2D payday game, but some persist and I am unable to find a workaround. Even selecting everything in the unit search menu doesn't select them. The biggest offenders with this are piles of paper and paper strewn across a desk. These are the little fuckers:
![20220620183320_1](https://user-images.githubusercontent.com/107096921/174562019-3f8315e5-90fb-48ca-a5fb-1c7d3042a95d.jpg)
![20220620183330_1](https://user-images.githubusercontent.com/107096921/174562032-43f87262-a29c-4732-a726-28800112e3b3.jpg)
Help :(

This also happens with filth on the walls. I've looked through all the decals and not found where or what the hell they are. This leads me to 2 conclusions:
1. A certain team of game developers couldn't agree on a singular naming scheme for the assets used in the game
2. The unit menu just kinda forgot their existence

**Comment by Oreztov** [comment, marked as answer]
These are special units (massunits) called brushes. You can access the appropriate menu by going to Main -> Brushes (next to AI), and there you'll find a clear all brushes trashcan.
Brushes are very useful for small details!

Oh, and good luck with that game you're working on :)

**Reply by AtomykAU** [reply]
Thank you so much, I had so many areas I couldn't screenshot because of this!
